Src/GoogleApis.Auth/OAuth2/Flows/GoogleAuthorizationCodeFlow.cs

```diff
@@ -35,11 +35,15 @@ public class GoogleAuthorizationCodeFlow : AuthorizationCodeFlow
         /// <summary>Gets the token revocation URL.</summary>
         public string RevokeTokenUrl { get { return revokeTokenUrl; } }
 
+        /// <summary>Gets or sets the include granted scopes indicator.</summary>
+        private bool? includeGrantedScopes { get; set; }

```

Comment:
You convinced me.
null -> nothing will sent on the wire
true/false -> the value will be sent as part of the URL.

That's what you meant, right?